Sources/AblyChat/DefaultMessages.swift

```diff
@@ -82,42 +82,53 @@ internal final class DefaultMessages: Messages {
             // (CHA-M5k) Incoming realtime events that are malformed (unknown field should be ignored) shall not be emitted to subscribers.
             let eventListener = channel.subscribe(RealtimeMessageName.chatMessage.rawValue) { [logger] message in
                 do {
+                    guard let action = MessageAction.fromRealtimeAction(message.action) else {
```

Comment:
we should never throw exception because of unknown type, we want to add new types whenever we want. It's something that we should always take into account regardless spec points. But in this case we have spec point:

```
*** @(CHA-M4m4)@ @[Testable]@ If a realtime message with an unknown @action@ is received, the SDK shall silently discard the message, though it may log at @DEBUG@ or @TRACE@ level.
```